### PR TITLE
Add redirects for constellation /get-started

### PR DESCRIPTION
**Pull request by irvinebroque**
PCX-8514

### CODE REVIEW

File: content/_redirects [hunk 1]
```diff
@@ -1114,6 +1114,7 @@
 # Constellation
 /constellation/platform/wrangler/ /constellation/ 301
 /constellation/platform/ /constellation/ 301
+/constellation/get-started/* /workers-ai/ 301
```
Comment (kodster28) [inline content/_redirects]: We need all the dynamic redirects at the bottom of the page. Otherwise, it breaks the static ones.

**Review by kodster28 — CHANGES_REQUESTED** [review]

File: content/_redirects (hunk 1 above)
Comment (irvinebroque) [reply content/_redirects]: Ah okay, will fix

Comment (kodster28) [reply content/_redirects]: Yeah, admittedly, the info in the pages docs isn't abundantly clear

Comment (kodster28) [reply content/_redirects]: Actually, @irvinebroque, I just pulled down your PR and moved it myself.

**Review by kodster28 — APPROVED** [review]

**Comment by kodster28** [comment]
also trying to reformat the redirects instructions in #11017

**Pull request merged**